1.0/docs/start/quick-tour.md

```diff
@@ -320,11 +320,13 @@ to the element's `owner` property, so it's updated as the user types.
   </div>
 </demo-tabs>
 
-**Note:** The `is="iron-input"` attribute indicates the input is a _type-extension_ custom 
-element; the element name is `iron-input`, and it _extends_ the native `<input>` element. 
+**Note:** The `is="iron-input"` attribute indicates the input is a 
+_type-extension_ custom element; the element name is `iron-input`, and it 
+_extends_ the native `<input>` element. To get this sample working you would 
```

Comment:
"To get this sample working" is a little confusing because it's working right there on screen. 
Maybe, "To run this sample locally, you'd need to install ... "